symfony/webpack-encore/0.9/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "devDependencies": {
+        "@symfony/webpack-encore": "^0.9.1"
+    },
+    "scripts": {
+        "dev-server": "./node_modules/.bin/encore dev-server",
```

Comment:
should just be `encore dev-server`, relying on the fact that NPM/Yarn adds the `.bin` folder in the PATH when running scripts.
This would make the NPM script portable on Windows (where writing the path with `/` does not work)